GetServiceByLatestRevision/__tests__/handler.test.ts

```diff
@@ -0,0 +1,120 @@
+// tslint:disable:no-any
+
+import { NonNegativeInteger } from "italia-ts-commons/lib/numbers";
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "italia-ts-commons/lib/strings";
+
+import {
+  NewService,
+  RetrievedService,
+  Service,
+  toAuthorizedCIDRs,
+  toAuthorizedRecipients
+} from "io-functions-commons/dist/src/models/service";
+
+import { MaxAllowedPaymentAmount } from "io-functions-commons/dist/generated/definitions/MaxAllowedPaymentAmount";
+import { ServicePublic } from "io-functions-commons/dist/generated/definitions/ServicePublic";
+
+import { none, some } from "fp-ts/lib/Option";
+import { fromLeft, taskEither } from "fp-ts/lib/TaskEither";
+import { NotificationChannelEnum } from "io-functions-commons/dist/generated/definitions/NotificationChannel";
+import { toCosmosErrorResponse } from "io-functions-commons/dist/src/utils/cosmosdb_model";
+import { aCosmosResourceMetadata } from "../../__mocks__/mocks";
+import { GetServiceByLatestRevisionHandler } from "../handler";
+
+afterEach(() => {
+  jest.resetAllMocks();
+  jest.restoreAllMocks();
+});
+
+const anOrganizationFiscalCode = "01234567890" as OrganizationFiscalCode;
```

Comment:
Can we move these mocks into `__mocks__/mocks.ts`, so they are available to other modules to test?